src/main/java/com/github/wovnio/wovnjava/Headers.java

```diff
@@ -24,6 +24,7 @@
 
     private final String requestLang;
     private final String clientRequestUrlWithoutLangCode;
+    private final String clientRequestPathWithoutLangCode;
```

Comment:
I think it is better to calculate with function instead of defining new member variable.
I want to reduce member variables as much as possible.

Reply:
Hm, I have been moving in the opposite direction, to be honest.

It feels easier to understand and reason around the state of the request handling when we simply compute the various state variables up front, and never allow them to change.

So I don't think we want to reduce member variables as much as possible. However, I do think this particular variable a good candidate to compute as needed, in the getter. Also, the variable is only used once, at least for now.

I'll make the change, thanks.